packages/react/src/Timeline/Timeline.module.css

```diff
@@ -0,0 +1,62 @@
+.Timeline {
+  --timelineSpacing: var(--base-size-16);
+  --itemLineHeight: var(--brand-text-lineHeight-300);
+  --bulletSize: var(--base-size-8);
+  --bulletSpacing: 2px;
+  --bulletY: calc(var(--itemLineHeight) / 2 - var(--bulletSize) / 2);
+  --lineWidth: 1.5px;
```

Comment:
Can we hoist these out into `@primer/brand-primitives` please, or do they need to be locally scoped?